bin/inference/pycbc_inference

```diff
@@ -136,7 +137,7 @@ if stilde_dict:
 with ctx:
 
     # read configuration file
-    cp = option_utils.config_parser_from_cli(opts)
+    cp = configuration.WorkflowConfigParser.from_clie(opts)
```

Comment:
@cdcapano Noticed this typo. `from_clie` --> `from_cli`

Reply:
:+1:

Reply:
Thanks for catching @soumide1102; fixed.